src/components/input/_macro.njk

```diff
@@ -47,7 +47,7 @@
         {% endif %}
     {% endset %}
     {% set field %}
-        {% if params.label or not (params.prefix or params.suffix or type == 'password') %}
+        {% if params.label is defined %}
```

Comment:
Surprised we can remove the other conditions. I've had a look on the netlify deployment and it looks ok!?

Reply:
Yeah I was too, I'm not sure about removing them because they had to have been put there for a reason. But it was the only way I could find of fixing the Jinja error. Just thought I'd create a PR to see what you thought. I'll see if I can find out why they were added.

Reply:
https://github.com/ONSdigital/design-system/pull/660/commits/cbbf1e50f2b8a04eb0a113440b8648d8bd2a6aa7 so the password bit was added in this commit. Ive tested the other stuff in that PR without the password bit and it seems to all work ok

Reply:
https://github.com/ONSdigital/design-system/pull/503/commits/84d1b82f40183e80a9da679fc0f4a943a399eb1b the prefix suffix stuff was added in this one. I've tested prefixed and suffixed inputs and they seem to work with this change too

Reply:
Maybe if you do a test yourself too and see if you can find any problems with it?